src/main/resources/filing/templates/small-full-accounts.html

```diff
@@ -1830,6 +1830,189 @@ <h2 class="text--center">Balance sheet</h2>
+
+                        <td id="total-assets-less-current-liabilities-curr-val" class="strong figure">
+                          {{if isNotEmpty $liabilities.current_total_assets_less_current_liabilities}}
+                              <ix:nonFraction contextRef="CY_END" decimals="0" unitRef="GBP" name="core:TotalAssetsLessCurrentLiabilities" format="ixt2:numdotdecimal">{{$liabilities.current_total_assets_less_current_liabilities}}</ix:nonFraction>
+                          {{end}}
+                        </td>
+
+                        <td id="total-assets-less-current-liabilities-prev-val" class="strong figure">
+                          {{if isNotEmpty $liabilities.previous_total_assets_less_current_liabilities}}
+                               <ix:nonFraction contextRef="PY_END" decimals="0" unitRef="GBP" name="core:TotalAssetsLessCurrentLiabilities" format="ixt2:numdotdecimal">{{$liabilities.previous_total_assets_less_current_liabilities}}</ix:nonFraction>
+                          {{end}}
+                        </td>
+                    </tr>
+
+
+                     {{if $creditors_after := $liabilities.creditors_amounts_falling_due_after_more_than_one_year}}
+                          <tr class="figures">
+                              <td id="creditors-after-label">Creditors: amounts falling due after more than one year:</td>
+
+                              <td class="figure noteIndex" id="uk-busCreditorsCreditorsAfterOneYear_CY_ENDNoteIndexes"></td>
+
+                              <td id="creditors-after-curr-val" class="strong figure">
+                                  {{if isNotEmpty $creditors_after.current_amount}}
+                                  {{if gt $creditors_after.current_amount 0.0}}({{end}}
+                                  <ix:nonFraction contextRef="CreditorsAfterOneYear_CY_END" decimals="0" unitRef="GBP" name="core:Creditors" format="ixt2:numdotdecimal">{{$creditors_after.current_amount}}</ix:nonFraction>
+                                  {{if gt $creditors_after.current_amount 0.0}}){{end}}
+                                  {{end}}
+                              </td>
+
+                              <td id="creditors-after-prev-val" class="strong figure">
+                                  {{if isNotEmpty $creditors_after.previous_amount}}
+                                  {{if gt $creditors_after.previous_amount 0.0}}({{end}}
+                                  <ix:nonFraction contextRef="CreditorsAfterOneYear_PY_END" decimals="0" unitRef="GBP" name="core:Creditors" format="ixt2:numdotdecimal">{{$creditors_after.previous_amount}}</ix:nonFraction>
+                                  {{if gt $creditors_after.previous_amount 0.0}}){{end}}
+                                  {{end}}
+                              </td>
+                          </tr>
+                      {{end}}
+
+                      {{if $provision := $liabilities.provision_for_liabilities}}
```

Comment:
Can we align this ` {{if $provision := $liabilities.provision_for_liabilities}}` block with the beginning of the the previous block `{{if $creditors_after := $liabilities.creditors_amounts_falling_due_after_more_than_one_year}}`?